azurelinuxagent/common/conf.py

```diff
@@ -19,11 +19,11 @@
 
 """
 Module conf loads and parses configuration file
-"""
+""" # pylint: disable=W0105
```

Comment:
does this affect the doctring?

Reply:
The error it is suppressing is:
`# pointless-string-statement<W0105>: (hi-pri) Used when a string is used as a statement (which of course has no effect). This is a particular case of W0104 with its own message so you can easily disable it if you're using those strings as documentation, instead of comments.`

I'm not quite sure what you mean by "affect[ing] the docstring," though.

Reply:
I wasn't sure if the doctring can have an inline comment. I just double-checked it can,

BTW - This one may be just easier to fix (make this a comment instead of a docstring) than to suppress and fix later

Reply:
all of these `W0105` were auto-marked by the script I wrote, which is part of the reason why they weren't fixed.

If you want, I can go through and fix them. Is this potentially beyond the scope of this PR, though? Since there are so many changes, I'd prefer to limit the number of changes that aren't just the insertion of a comment.

Reply:
sure, you can leave them as is.